### ISSUE TITLE
'Failed to add a video capture filter.': Value does not fall within the expected range

### ISSUE THREAD

**Issue by simonallen5** [jacobbo/WebEye#49]
As a new c# programmer I am trying to integrate the webcamera control into an existing program. I have )I think) installed all the required bits - and the demo program runs fine on my pc. However I am getting the 'Failed to add a video capture' error when implemented within my own program, with the 'Inner Exception' ArgumentException: Value does not fall within the expected range.

Any help much appreciated!

**Comment by jacobbo** [comment]
Hello, here are the sources of demo apps:
https://www.codeproject.com/KB/audio-video/330177/WpfSource.zip
https://www.codeproject.com/KB/audio-video/330177/WinFormsSource.zip

**Comment by simonallen5** [comment]
That's what I thought! I had copied the code elements from the Demo App into my app - it compiles OK but I get the above error on clicking the Start button. I'll try it the other way round - import my code into the Demo App, which is working,  and see what happens

**Comment by jacobbo** [comment]
If you start a new solution, then you have to add a Nuget package by running the following command in the Nuget console: Install-Package WebEye.Controls.Wpf.WebCameraControl

**Comment by simonallen5** [comment]
Yes - did that. I was able to see the dll and put the control into the Toolbox, from where I added it to my form
